lib/contract.cu

```diff
@@ -73,5 +245,5 @@ public:
     errorQuda("Contraction code has not been built");
   }
 #endif
-
+  //----------------------------------------------------------------------------
```

Comment:
I don't think this line serves much purpose?

Reply:
removed in c2d6e23